.gitlab/e2e.yml

```diff
@@ -124,7 +124,7 @@ k8s-e2e-otlp-main:
 
 .new_e2e_template:
   stage: e2e
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner-dev:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
```

Comment:
TODO: Remove and update `test-infra-definitions` again once DataDog/test-infra-definitions#380 is merged.